test/unit/main/interactive_shell.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
```

Comment:
We've discussed it offline and the best place for this test would be in `test/system`. Could you move it there and modify the corresponding CMakeLists.txt file to add the test? It should be something like:

```
add_test(interactive_shell interactive_shell.py)
```

Reply:
Done!